### DISCUSSION TITLE
How to Add a Directory as Context?

### DISCUSSION THREAD

**Discussion by WIZARDELF** [karthink/gptel#432] [Q&A]
Is it possible to send a directory of files at one time?

**Comment by karthink** [comment]
You can open the directory in dired, mark all files with `t`, then run `M-x gptel-add`.

**Reply by Lyonsclay** [reply]
I used `m` to mark the files.

**Reply by karthink** [reply]
Actually gptel has supported adding directories for a while now -- you can add them the same way as a regular file.

**Reply by Lyonsclay** [reply]
Ah that's great; I hadn't thought to try it.

**Reply by chookity-pokk** [reply]
Would it be worth changing the name of `gptel-add-file` or the description to note that it can also add directories? Right now it says "Add the file at PATH to gptel as context".
